challenges/objects-arrays.js

```diff
@@ -59,11 +87,29 @@ Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
-console.log(contactInfo);
+
+for (let index = 0; index < graduates.length; index++) {
+  contactInfo.push(graduates[index]['first_name']);
+  contactInfo.push(graduates[index]['email']);
+  contactInfo.push(',')
+
+}
+
+console.log(contactInfo.join(' '));
```

Comment:
Interesting execution! Perfectly fine, just another way to go about it: 

const contactInfo = [];
for(let i = 0; i < graduates.length; i++) {
  contactInfo.push(graduates[i].first_name + " " + " " + graduates[i].email);
}
console.log(contactInfo);